package.json

```diff
@@ -34,7 +34,7 @@
     "release:validate": "./scripts/validateRelease.sh",
     "db:init": "FORCE_DROP=true node ./scripts/sync_models",
     "db:init:memeo:inmem": "FORCE_DROP=true node ./scripts/sync_models && sequelize db:seed --seed memeolist-example-inmem.js",
-    "db:init:memeo:postgres": "FORCE_DROP=true node ./scripts/sync_models && sequelize db:seed --seed memeolist-example-postgres.js && docker exec aerogeardatasyncserver_postgres_memeo_1 psql -U postgres -d memeolist_db -f /tmp/examples/memeolist.tables.sql",
+    "db:init:memeo:postgres": "FORCE_DROP=true node ./scripts/sync_models && sequelize db:seed --seed memeolist-example-postgres.js && docker exec datasyncserver_postgres_memeo_1 psql -U postgres -d memeolist_db -f /tmp/examples/memeolist.tables.sql",
```

Comment:
If we're changing the pod name here, we also need to update the docker compose command (currently `docker-compose -p aerogeardatasyncserver up -d`). I think docker compose command is only mentioned in README.

By default, docker compose prefixes the pod names with the directory name unless `-p` param is used. I think you had a different directory name and you didn't use `-p` and you had trouble.

Reply:
2nd thought: let's revert this change :)